app/controllers/infra_networking_controller.rb

```diff
@@ -430,7 +437,7 @@ def set_right_cell_vars
         x_history_add_item(:id => x_node, :text => header, :action => @sb[:action])
       end
     end
-    action = nil
+    action = @sb[:action] == "button" ? "dialog_form_button_pressed" : nil
```

Comment:
This looks like you're assuming any kind of button is a dialog button.

I guess it's true since most buttons are x_button in explorers .. but.. is there a better way you can tell?

(If not, not, but if we decided to unify button actions between explorers and non-explorers, this would break.)